types/paradisebotsapi.js/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for paradisebotsapi.js 1.0
+// Project: https://github.com/ParadiseBotList/paradisebotsapi.js#readme
+// Definitions by: Yoshida Tomio <https://github.com/1chiSensei>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare class PBL {
+    constructor(id?: string, auth?: string);
+
+    post(server_count: number, shard_count: number): Promise<void>;
+    get(botID: string, response: (data: Record<string, unknown>) => void): Promise<void>;
+    getUser(userID: string, response: (data: Record<string, unknown>) => void): Promise<void>;
+}
+
+export = PBL;
```

Comment:
This export shape appears to be new in [1.0.6](https://github.com/ParadiseBotList/paradisebotsapi.js/commit/2bb64206e18ba3d62623e5188eb60a308ff1c3e6#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346), but the newest published version seems to be [1.0.1](https://unpkg.com/browse/paradisebotsapi.js@1.0.1/index.js).

Reply:
There are multiple packages [paradiseapi.js](https://npmjs.com/package/paradiseapi.js) and [paradisebotsapi.js](https://npmjs.com/package/paradisebotsapi.js)

Reply:
ill be changing it in a while i have classes

Reply:
No rush. 😄   I'm not sure I understand though - these appear to be the types for paradisebotsapi.js, not paradiseapi.js.

Reply:
Yep. But they have the same repo so I'll use unpkg so I can get the correct source.

Reply:
my local NPM installs `"paradiseapi.js@1.10.0"`, not `1.0.1`

Reply:
Its `paradisebotsapi.js` not `paradiseapi.js`